draft-ietf-ppm-dap.md

```diff
@@ -867,23 +867,29 @@ with error of type 'outdatedConfig'. If the leader supports multiple HPKE
 configurations, it can use trial decryption with each configuration to determine
 if requests match a known HPKE configuration. When clients receive an 'outdatedConfig'
 error, they SHOULD invalidate any cached aggregator `HpkeConfigList` and retry with
-a freshly generated Report. If this retried report does not succeed, clients MUST
-abort and discontinue retrying.
+a freshly generated Report. If this retried report does not succeed, clients
+SHOULD abort and discontinue retrying.
+
+If a report's metadata matches that of a previously uploaded report, the leader
+MUST ignore it and the leader MAY alert the client with error `reportRejected`.
 
 The Leader MUST ignore any report pertaining to a batch that has already been
-collected. (See {{input-share-validation}} for details.) Otherwise,
-comparing the aggregate result to the previous aggregate result may result in a
-privacy violation. (Note that the Helpers enforce this as well.) The Leader MAY
-ignore any reports whose timestamp is past the task's `task_expiration`. When it
-does so, the leader SHOULD abort the upload protocol and alert the client with
-error "reportTooLate". Client MAY choose to opt out of the task if its own clock
-has passed `task_expiration`.
+collected (see {{input-share-validation}} for details). Otherwise, comparing the
+aggregate result to the previous aggregate result may result in a privacy
+violation. Note that the Helpers enforce this as well. In addition, the leader
```

Comment:
In case we end up hard-coding two aggregators in the future.
```suggestion
violation. Note that this is enforced by all Aggregators, not just the Leader. In addition, the leader
```